style.css

```diff
@@ -0,0 +1,119 @@
+/* For mobile phones */
+@media only screen and (max-width: 600px) {
+  * {
+    box-sizing: border-box;
+    margin: 0;
+  }
+
+  body {
+    background-color: #1c1a19;
+    height: 50vh;
+    background-image: url("./Images/Illustration.svg");
+    background-repeat: no-repeat;
+    background-position: 100% -340%;
```

Comment:
- [ ] Please fix the background image , it is currently not displaying. I would suggest changing `background-position: 100% -340%` to `background-position: 100% 220%`  Cheers 👍

Reply:
@sumairq 
when i change the background-position to 100%, 220%, this is how it looks on my display
<img width="534" alt="Screenshot 2022-08-18 at 9 28 09 AM" src="https://user-images.githubusercontent.com/106558709/185362009-d0809a69-b2f0-461a-b8e3-6704a9f1def6.png">

So the -340% makes it look like this on my screen: 
<img width="438" alt="Screenshot 2022-08-18 at 9 34 59 AM" src="https://user-images.githubusercontent.com/106558709/185363023-4eb0236d-eb3b-4e21-9382-dec22a1280c3.png">

Reply:
Any suggestions please @sumairq